src/lib/userStorage/UserProfileStorage.js

```diff
@@ -388,17 +395,36 @@ export class UserProfileStorage implements ProfileStorage {
         {},
       )
 
-    return publicProfile
+    const { fullName, smallAvatar } = publicProfile
+
+    logger.info(`getPublicProfile by field <${field}>`, { fullName })
+
+    return { name: fullName, avatar: smallAvatar }
   }
 
+  /**
+   * Returns ProfileField value. It may be encrypted
+   * @param field
+   * @returns {EncryptedField}
+   */
   getProfileFieldValue(field: string): string {
     return this.profile[field]?.value
   }
 
+  /**
+   * Returns profile field display value
+   * @param field
+   * @returns {string}
+   */
   getProfileFieldDisplayValue(field: string): string {
     return this.profile[field]?.display
   }
 
+  /**
+   * Returns ProfileField
+   * @param field
+   * @returns {ProfileField}
```

Comment:
doesnt match return type

Reply:
changed in d65c060a3868c252b0641e1aa2f9c28d663e1efa